statsmodels/tsa/vector_ar/tests/JMulTi_results/parse_jmulti_var_output.py

```diff
@@ -0,0 +1,421 @@
+import re
```

Comment:
Haha, I know that 208 files look like a lot of work for reviewing. Sorry for that! I thought that it would be the most transparent way to add the files generated by JMulTi directly to statsmodels. I will think about how I could accomplish this and appreciate any hints on that!
I am afraid that rewriting the program correspondingly would mean a lot of work too :smile: So if it's possible I would kindly ask you to ignore the high number of files. After all, the `.txt` files in `.../vector_ar/tests/JMulTi_results` and in `.../vecm/tests/JMulTi_results` constitute the majority of files and can safely be skipped when reviewing.